rules/Proj1101.md

```diff
@@ -0,0 +1,36 @@
+﻿# Proj1101: Package references should have stable versions
+When building code that goes into production, your code should not rely on a
```

Comment:
Code that goes into production should not rely on a nightly build or a prerelease version of a package.